src/components/MapController/DualMapController/DualMapController.js

```diff
@@ -61,21 +80,10 @@ export default class MapController extends React.Component {
   }
 
   // Support functions
-  
-  hasValidData(symbol, props = this.props) {
-    var dataLocation = symbol === 'variable' ? 'meta' : 'comparandMeta';
-
-    return !_.isUndefined(props[dataLocation]) &&
-      props[dataLocation].length > 0;
-  }
-
-  hasComparand() {
-    return this.hasValidData('comparand');
-  }
 
   // TODO: https://github.com/pacificclimate/climate-explorer-frontend/issues/118
-  currentDataset() {
-    // Return encoding of currently selected dataset
+  currentInstance() {
```

Comment:
Yeah, I've never really liked the use of "instance". Instance usually refers to *one* thing, so using it to describe a set of selecting filters that could describe multiple sets of data doesn't really match.

Reply:
I think datasetSpec would be confusing, since in the context of of ce frontend, "dataset" means "a single file with data in it" and this specification doesn't uniquely describe a single file. It uniquely describes a single - I don't know what to call it, which is sort of how it got called instance - chunk of model output, either a single run or a single run and climatology period, but there can be multiple files representing different views of that output and the specification describes all of them.

How do you feel about just dataSpec?

Alternately, we could call single files datafiles and "set of files representing a specific data chunk and / or the parameters that uniquely describe that chunk" datasets, they'd be a set of files.

Reply:
dataSpec is just fine with me.

Reply:
dataSpec works for me.